src/script/view_model/MainViewModel.js

```diff
@@ -60,15 +57,8 @@ z.viewModel.MainViewModel = class MainViewModel {
   }
 
   constructor(repositories) {
-    this.closePanel = this.closePanel.bind(this);
```

Comment:
All those functions do not need to be manually bound (as they are always consumed directly `view.method()` and never given `const method = view.method`).
(expect for `togglePanel` and `closePanelOnClick`, for them I used the `method = () => {}` technique (in order to cleanup the constructor)